helper/diffPlaces.js

```diff
@@ -59,6 +64,13 @@ function isParentHierarchyDifferent(item1, item2){
   // note: this really shouldn't happen as at least one parent should exist
   if( !isPojo1 || !isPojo2 ){ return false; }
 
+  // US states should not be deduplicated, except against other US states
```

Comment:
trying to wrap my head around this, is it an XOR?

Reply:
there is another function called `isLayerDifferent` which handles figuring out which layers can be considered synonymous, maybe it makes more sense to have this functionality in there? as like a subset of the includes check?